controllers/admin/AdminLinkWidgetController.php

```diff
@@ -218,11 +219,45 @@ protected function buildHelper()
         return $helper;
     }
 
-    public function validateForm($data)
+    /**
+     * @param array $fields
+     * @return bool
+     */
+    public function validateLinkName(array $fields)
```

Comment:
why do we even need this here? If field is required it would be automatically copied from one language to another and if field is empty ObjectModel will throw an error